Resources/installer/sql/demo_clean_data.sql

```diff
@@ -265,36 +265,36 @@ UNLOCK TABLES;
-INSERT INTO `ezcontentclass` VALUES (0,'<title>',1422896964,14,19,'call_to_action',2,1,3,1422896964,14,'5ef96fd48e645dbdeb803014fe7af178','a:2:{s:6:\"eng-GB\";s:0:\"\";s:16:\"always-available\";s:6:\"eng-GB\";}','a:2:{s:6:\"eng-GB\";s:14:\"Call To Action\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
-INSERT INTO `ezcontentclass` VALUES (0,'<title>',1422896964,14,20,'call_to_action_feedback',2,0,3,1422896964,14,'d88d2cd52ba6c6b1383f661a959c995f','a:2:{s:6:\"eng-GB\";s:0:\"\";s:16:\"always-available\";s:6:\"eng-GB\";}','a:2:{s:6:\"eng-GB\";s:23:\"Call To Action Feedback\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
-INSERT INTO `ezcontentclass` VALUES (0,'<name>',1422896964,14,21,'product',2,0,3,1422896964,14,'77f3ede996a3a39c7159cc69189c5307','a:1:{s:6:\"eng-GB\";s:0:\"\";}','a:2:{s:6:\"eng-GB\";s:7:\"Product\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
-INSERT INTO `ezcontentclass` VALUES (0,'<name>',1422896964,14,22,'feedback_form',2,1,3,1422896964,14,'df0257b8fc55f6b8ab179d6fb915455e','a:1:{s:6:\"eng-GB\";s:0:\"\";}','a:2:{s:6:\"eng-GB\";s:13:\"Feedback form\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
-INSERT INTO `ezcontentclass` VALUES (0,'<name>',1422896964,14,23,'landing_page',2,1,3,1422896964,14,'e36c458e3e4a81298a0945f53a2c81f4','a:1:{s:6:\"eng-GB\";s:0:\"\";}','a:2:{s:6:\"eng-GB\";s:12:\"Landing Page\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
-INSERT INTO `ezcontentclass` VALUES (0,'<title>',1422896964,14,24,'wiki_page',2,1,3,1422896964,14,'d4a05eed0402e4d70fedfda2023f1aa2','a:1:{s:6:\"eng-GB\";s:0:\"\";}','a:2:{s:6:\"eng-GB\";s:9:\"Wiki Page\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
-INSERT INTO `ezcontentclass` VALUES (0,'<name>',1422896964,14,25,'poll',2,0,3,1422896964,14,'232937a3a2eacbbf24e2601aebe16522','a:0:{}','a:2:{s:6:\"eng-GB\";s:4:\"Poll\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
+INSERT INTO `ezcontentclass` VALUES (0,'<short_title|title>',1422896964,14,16,'article',2,1,3,1440410165,14,'c15b600eb9198b1924063b5a68758232','a:1:{s:6:\"eng-GB\";s:164:\"Article is to be used for storing single-page articles. Note: be careful, don\'t use it for storing a case study, use the dedicated Case Study content type for that.\";}','a:2:{s:6:\"eng-GB\";s:7:\"Article\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
+INSERT INTO `ezcontentclass` VALUES (0,'<name>',1422896964,14,17,'blog',2,1,3,1440409131,14,'3a6f9c1f075b3bf49d7345576b196fe8','a:1:{s:6:\"eng-GB\";s:252:\"Defines a structure for storing a blog (short articles by a single person and/or on a particular topic.) Don\'t forget! there are three key elements here: Tag cloud, Description of the Blog and Alphabetical list of all the tags for all individual posts.\";}','a:2:{s:6:\"eng-GB\";s:4:\"Blog\";s:16:\"always-available\";s:6:\"eng-GB\";}',1,1,'',0);
```

Comment:
" Don\'t forget! there are three key elements here: Tag cloud, Description of the Blog and Alphabetical list of all the tags for all individual posts."

Maybe drop this as it depends on templates, and secondly I don't get the last part :P